src/test/java/uk/gov/hmcts/cmc/claimstore/services/notifications/BaseNotificationServiceTest.java

```diff
@@ -32,11 +37,10 @@
     protected static final String USER_EMAIL = "user@example.com";
     protected static final String USER_FULLNAME = "Steven Patrick";
 
-    private PrintStream systemOut;
-    private ByteArrayOutputStream outContent;
-
     protected final Claim claim = SampleClaim.getDefault();
 
+    protected Logger log = (Logger) LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME);
```

Comment:
Are these changes related to the updates?

Reply:
yep broken with logging lib update